open-apparel-registry/open-apparel-registry
# Enable hiding sectors in embed

## Overview

Adds a check box to the Embedded Map settings page that allows a user to toggle the display of the sector field on and off.

Connects #2187 

## Demo

<img width="1437" alt="Screen Shot 2022-10-04 at 1 45 46 PM" src="https://user-images.githubusercontent.com/21046714/193889774-e5b66f62-b854-4c4a-aef1-84c1151e7f36.png">
<img width="1365" alt="Screen Shot 2022-10-04 at 1 46 01 PM" src="https://user-images.githubusercontent.com/21046714/193889795-02ffaaa7-4f53-4d62-9b60-3204fd0ab0c5.png">
<img width="759" alt="Screen Shot 2022-10-04 at 1 47 04 PM" src="https://user-images.githubusercontent.com/21046714/193889926-5f4ed14f-b933-47c9-94cd-0964cd762e48.png">

## Testing Instructions

* Run `./scripts/manage migrate` and `./scripts/server`
* Login as c2@example.com and visit the embed tab in the settings. Confirm that the 'Hide sectors' checkbox is present. 
* Login as c3@example.com and visit the embed tab in settings. Confirm that the 'Hide sectors' checkbox is present (c3 has a lower embed permission level than c2). 
* Give the embedded map a height/width, and confirm that the sector filter is shown on the search page, and the sector field is shown on the facility details page. 
* Check the box for 'Hide sector data'. Confirm that the sector filter is hidden on the search page, and the sector field is hidden on the facility details page. 

## Checklist
